src/GraphQL.Benchmarks/Properties/launchSettings.json

```
@@ -1,8 +1,7 @@
 {
   "profiles": {
     "Profiler": {
-      "commandName": "Project",
-      "commandLineArgs": "test"
+      "commandName": "Project"
```

```suggestion
      "commandName": "Project",
      "commandLineArgs": "test"
```

The point is to run the profiler, not benchmarks. If you wish then add second profile like "Bench" without args.